### PR TITLE
Feature/rpnv1 948 remove version serialization

### PR DESCRIPTION
**Pull request by clstrfsck**

### CODE REVIEW

File: radixdlt-core/radixdlt/src/test/java/org/radix/GenerateUniversesTest.java [hunk 1]
```diff
@@ -172,7 +172,7 @@ public void testJson() throws IOException {
 		}
 		assertThat(err).isEmpty();
 		assertThat(out)
-			.hasLineCount(278)
+			.hasLineCount(242)
```
Comment (clstrfsck) [inline radixdlt-core/radixdlt/src/test/java/org/radix/GenerateUniversesTest.java:175]: That is quite a lot of version properties that are gone now.

**Review by talekhinezh — APPROVED** [review]

**Review by Sajjon — APPROVED** [review]
LGTM

**Review by siy — APPROVED** [review]
LGTM

**Comment by clstrfsck** [comment]
Will be merged in #19

**Pull request closed without merging**